conda/core/prefix_data.py

```diff
@@ -391,11 +392,11 @@ def _get_json_fn(self, prefix_record: PrefixRecord) -> str:
         return fn + ".json"
 
     def insert(self, prefix_record: PrefixRecord, remove_auth: bool = True) -> None:
-        assert prefix_record.name not in self._prefix_records, (
-            f"Prefix record insertion error: a record with name {prefix_record.name} already exists "
-            "in the prefix. This is a bug in conda. Please report it at "
-            "https://github.com/conda/conda/issues"
-        )
+        if prefix_record.name in self._prefix_records:
+            raise CondaError(
+                f"Prefix record insertion error: a record with name {prefix_record.name} already exists "
+                "in the prefix. This can often be resolved by running `conda clean --all`. "
```

Comment:
Is `conda clean --all` actually fixing this? I don't see anything in the code where it cleans duplicates in `conda-meta` 🤔

Reply:
This needs to be verified thoroughly if we are going to include it in the error message.